userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -292,6 +274,36 @@ of the following:
 [wordmark]: https://en.wikipedia.org/wiki/Wordmark
 [your logo]: /docs/adding-content/iconsimages/#add-your-logo
 
+### Light/Dark mode menu
+
```

Comment:
I'd add a little intro sentence just in case users aren't familiar with what this would actually do

"If you enable this feature, Docsy adds a menu item to your navbar that lets users switch your site's text display between a default "light" mode, and a "dark" mode where the text is displayed in a light color on a dark background."

Reply:
I've added the text you suggested.